change/@azure-msal-browser-2020-10-21-13-05-39-msal-browser-storage-update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
Does this add features to the public API? Otherwise can be a patch.

Reply:
Made it a patch!